scripts/enter-env.sh

```diff
@@ -32,7 +32,7 @@ if [ ! -z "$SETTINGS_FILE" -o ! -z "$XILINX" ]; then
 fi
 
 # Check ixo-usb-jtag *isn't* install
-if [ -d /lib/firmware/ixo-usb-jtag/ ]; then
+if [ ! -d /lib/firmware/ixo-usb-jtag/ ]; then
```

Comment:
?

Reply:
Because I always have this dirty in my checkout. Sorry.